src/main/java/com/rtb/projectmanagementtool/user/AuthOps.java

```diff
@@ -0,0 +1,46 @@
+package com.rtb.projectmanagementtool.user;
```

Comment:
This would be a new feature that's related to User, but not exactly the same as User.  It would be good to put this in its own package.  Also naming wise, I would suggest "AuthHelper" or "AuthUtils".  "AuthOps" is also good, but the former 2 are more standard and more flexible if you need to add a non-operation method.